lms/resources/_js_config/__init__.py

```diff
@@ -83,6 +84,10 @@ def add_document_url(self, document_url):
                 "launchUrl": launch_url,
                 "launchParams": launch_params,
             }
+        elif self._jstor_service.enabled and document_url.startswith("jstor://"):
```

Comment:
Added the `self._jstor_service.enabled` check here.

`document_url.startswith("jstor://")` could be in `_jstor_service.is_jstor` or similar but is simple enough that is easier to read it inline.